<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersusView</class>
 <widget class="QWidget" name="VersusView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="mainLayout" stretch="15,3">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="kanal_chart_view">
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter_a">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="handleWidth">
         <number>2</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <layout class="QVBoxLayout" name="vl_wave"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_vew_set">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="15" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Y-max</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>X-axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QComboBox" name="cb_sat_x_wave">
          <item>
           <property name="text">
            <string>ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>deg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>points</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Waveform :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Setting Kanal:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Data Type</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cb_data_tipe">
          <item>
           <property name="text">
            <string>Acceleration</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Velocity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Displacement</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Accel. Unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="cb_sat_accel">
          <item>
           <property name="text">
            <string>G</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mm/s2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Velo. Unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="cb_sat_velo">
          <item>
           <property name="text">
            <string>mm/s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IPS</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Disp. Unit</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="cb_sat_disp">
          <item>
           <property name="text">
            <string>mm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>micron</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mils</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QComboBox" name="cb_ymax">
          <item>
           <property name="text">
            <string>0.01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100000</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="20" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>view length</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QComboBox" name="cb_wave_length">
          <property name="currentIndex">
           <number>5</number>
          </property>
          <item>
           <property name="text">
            <string>10%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50%</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100%</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Channel-1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="check_kanal1">
          <property name="text">
           <string>view</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Channel-2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="check_kanal2">
          <property name="text">
           <string>view</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Channel-3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Channel-4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="check_kanal3">
          <property name="text">
           <string>view</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="check_kanal4">
          <property name="text">
           <string>view</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Windowing</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="cb_window">
          <item>
           <property name="text">
            <string>Hanning</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rectangular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hamming</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
